Remotion/ObjectBinding/Web.Development.WebTesting/ControlObjects/BocReferenceValueControlObject.cs

```diff
@@ -218,6 +219,11 @@ public IReadOnlyList<string> GetValidationErrors ()
       return GetValidationErrors (GetValueScope());
     }
 
+    public IReadOnlyList<string> GetValidationErrorsForReadOnly ()
+    {
+      return GetValidationErrorsForReadOnly (GetCommand().Scope);

```

Comment:
In z230 we use Scope.FindChild("Command"), here we use GetCOmmand().Scope. Is there a consistent way to do this?